flocker/node/_loop.py

```diff
@@ -379,21 +379,12 @@ class ConvergenceLoopOutputs(Names):
     [_FIELD_CONNECTION, _FIELD_LOCAL_CHANGES], [],
     u"Send the local state to the control service.")
 
```

Comment:
:question: Shall we add logging changes to a separate PR and create followup issues for adding the logging back, but with limits on number of datasets and containers logged. I think we still get a log of the updated configuration in the flocker-control logs, right? And when DeploymentState changes. So it shouldn't hinder our debugging too badly while we figure out the right thing to do.